qubesbuilder/plugins/installer/conf/comps-dom0.xml

```diff
@@ -0,0 +1,1488 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This one is in meta-packages submodule, no need to copy here.

Reply:
Yes, I don't remember why it has fallen here.